### ISSUE TITLE
A 'move' function

### ISSUE THREAD

**Issue by firatsarlar** [facebook/immutable-js#993]
A move function would be nice.
I've tested something like this : 

```
Immutable.List.prototype.move = function( fromIndex, toIndex){


  return this.splice(fromIndex, 1)
          .insert(toIndex, this.get(fromIndex));

}
```

**Comment by leebyron** [comment]
Closing this aging issue.

I'd like to avoid an ever-growing prototype of methods. Since the implementation for this is quite small, it looks like it would be a reasonable function in your application instead of in the data structure library

**Comment by firatsarlar** [comment]
I did not propose one of ever-growing prototype of methods you avoid.
The thing I wrote is a proof of concept, working function , no more , no less.

Is there any laws - like Newton's - that states, a data structure should not have a move method,
or is this your opinion - or general opinion current dev. world we perceive - ?

Let me explain my use case, - there was some code back then, I could not find it -,
A list of pictures, you drag - drop them to change their order ,
and finally show them to end user the order you wish to.
